src/template.rs

```diff
@@ -593,7 +604,29 @@ mod tests {
             ),
         );
 
-        assert_eq!(render!("{{chains.chain1}}", context).unwrap(), "hello!");
+        assert_eq!(render!("{{chains.chain1}}", context).unwrap(), expected);
+    }
+
+    #[tokio::test]
+    async fn test_chain_command_stdin_chain_nested_error() {
```

Comment:
Could you make this a test case on `test_chain_command_error` instead? You can just add a `stdin` param to the test for it. It'll reduce boilerplate and keep the related tests together. For the name, I think `case::stdin_error` is fine

Reply:
Done